### DISCUSSION TITLE
No such file or directory

### DISCUSSION THREAD

**Discussion by DimaKozhanovsky** [open-tutorials/cypress#59] [Q&A] [answered]
Colleagues, I'm doing a tutorial "Deploy project Conduit locally"  third paragraph https://www.epic1h.com/build_conduit. Below are described steps and peculiarities of this issue on my own laptop .

My expectations, this is a clone of the "Conduit" project, installed locally.

1. Open the default terminal on Mac.
2. Make sure that you are in the appropriate folder (in my case it is  empty folder "untitled folder")
3. Execute command from this folder smiller@Mac-mini-Dima untitled folder
%` git@github.com:DimaKozhanovsky/conduit-realworld-example-app.git `

**Error message is displyed** 
`zsh : no such file or directory: git@github.com:DimaKozhanovsky/conduit-realworld-example-app.git`

**Please , help me to find out  the reason of issue ? Thank you in advance 🙏**

<img width="851" alt="Error" src="https://user-images.githubusercontent.com/74324758/219110523-a6602de2-60f3-448c-99d7-7534e8ddfc77.png">

**Comment by breslavsky** [comment]
@Vladimir4932 please help to @DimaKozhanovsky

**Reply by breslavsky** [reply]
@DimaKozhanovsky could you please attach screenshot?

**Reply by Vladimir-Tarasov** [reply]
> @Vladimir4932 please help to @DimaKozhanovsky

Ok. I will try.

**Comment by breslavsky** [comment]
@DimaKozhanovsky please fix description like in example https://github.com/open-tutorials/cypress/discussions/11

**Comment by DimaKozhanovsky** [comment]
@breslavsky Please check it out now , do you have any comments for improvement ?

**Reply by breslavsky** [reply]
Yes, I have.

- [ ] You have broken spaces.
- [ ] You have no dots at the end sentences.
- [ ] You can wrap code to the next line.

<img width="600" border=1 alt="screen" src="https://user-images.githubusercontent.com/8783224/219130616-ee0f58d6-80f4-4e39-bd10-0e0c84002131.png">

**Comment by Vivioza5** [comment, marked as answer]
`git@github.com:DimaKozhanovsky/conduit-realworld-example-app.git` 

it is wrong command. You use git, if you type only this command, you have result such as you see.

right commans it's:

`git clone git@github.com:DimaKozhanovsky/conduit-realworld-example-app.git`

**Reply by Vivioza5** [reply]
It [was written](https://github.com/open-tutorials/cypress/blob/2f423e6cee6f4202b7d991627238e2a47de2999e/tutorials/build_conduit.md?plain=1#L198) in the tutorial.

**Reply by breslavsky** [reply]
Where?

**Reply by Vladimir-Tarasov** [reply]
The right command is written in the tutorial.
